addon/ajax-request.js

```diff
@@ -47,6 +48,14 @@ export default class AjaxRequest {
       type: hash.type,
       url: hash.url
     };
+
+    const allHeaders = this._getFullHeadersHash(hash.headers);
+    if (allHeaders['Content-Type'] === JSONAPIContentType) {
```

Comment:
Huh, turns out that `.startWith` requires the Babel polyfill.  I didn't realize that.